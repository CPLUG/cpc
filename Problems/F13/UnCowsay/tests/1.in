 _____________________________________________________________________
< "I'd love to go out with you, but my favorite commercial is on TV." >
 ---------------------------------------------------------------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
