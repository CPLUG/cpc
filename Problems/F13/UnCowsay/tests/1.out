"I'd love to go out with you, but my favorite commercial is on TV."
